<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionTableViewWidiget</class>
 <widget class="QWidget" name="optionTableViewWidiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="bevelButton">
     <property name="text">
      <string>Bevel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="optionTableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
